aodncore/util/fileops.py

```diff
@@ -137,7 +138,7 @@ def is_dir_writable(path):
     """
     try:
         with TemporaryFile(prefix='is_dir_writable', suffix='.tmp', dir=path) as t:
-            t.write('is_dir_writable')
+            t.write('is_dir_writable'.encode('utf-8'))
```

Comment:
Is `'x'.encode('utf-8')` the same as `u'x'`?

Reply:
It actually needs a "bytes-like" object not a unicode. Could have been written as a b-string, but results in the same thing.

```python
In[20]: b'is_dir_writable' == 'is_dir_writable'.encode('utf-8')
Out[20]: True
```

Reply:
Ah! Right, thanks. Still getting my head around bytes/string/unicode objects.